javascript/javascript2/week1/spiritAnimalNameGenerator.js

```diff
@@ -0,0 +1,54 @@
+
+const spiritAnimalNameList = [
+    'Sensible Fuchsia Mocking Bird',
+    'Steady Plumboa',
+    'Purple Cottontail',
+    'Contemplative Tea Beetle',
+    'Big eyes rhinos',
+    'the fullmoon wolf',
+    'Cute blue cat',
+    'the sharp shark',
+    'long tailed monkey',
+    'so cute white dove'
+];
+
+function spiritAnimalNameGenerator(name) {
+    return `${spiritAnimalNameList[parseInt(Math.random() * 10)]}`;
```

Comment:
Since you're only using the spiritAnimalNameList in this method, I'd put it in here.

Reply:
i did the same way , but I did not understand what you meant to say?